airbyte-integrations/connectors/source-square/source_square/spec.yaml

```diff
@@ -75,17 +75,6 @@ connectionSpecification:
       title: Include Deleted Objects
       default: false
       order: 3
-authSpecification:
```

Comment:
I have the same question as [this other PR](https://github.com/airbytehq/airbyte/pull/26358/files#r1204215234)